lib/types/beacon_chain/fork.ex

```diff
@@ -18,4 +20,13 @@ defmodule Types.Fork do
           current_version: Types.version(),
           epoch: Types.epoch()
         }
+
+  @impl LambdaEthereumConsensus.Container
+  def schema do
+    [
+      {:previous_version, {:int, 4}},
+      {:current_version, {:int, 4}},
```

Comment:
We need a way to alias types. I propose the following solution:
1. Add a `type_aliases.ex` file/module.
2. In it, add functions, like `version()` that return the aliased schema.
3. In files like this, we just `import TypeAliases` and we can just call `version()` here afterwards. 

We can do that with all of the aliases in the table that is present in this section of the eth2book: https://eth2book.info/capella/annotated-spec/#custom-types